src/components/Chat/ModelsDialog.tsx

```diff
@@ -277,17 +269,17 @@ export const ModelsDialog: FC<Props> = ({
               onChange={(e) => {
                 handleSearch(e.target.value);
               }}
-              className="m-0 w-full rounded border border-gray-400 bg-transparent px-3 py-2 outline-none placeholder:text-gray-500 focus-visible:border-blue-500 dark:border-gray-600 dark:focus-visible:border-blue-500"
+              className="m-0 w-full rounded border border-primary bg-transparent px-3 py-2 outline-none placeholder:text-secondary focus-visible:border-accent-primary"
             ></input>
           </div>
 
           <div className="flex gap-2 px-3 md:px-5">
             <button
               className={classNames(
-                'rounded border-b-2 px-3 py-2 hover:bg-blue-500/20 dark:hover:bg-blue-500/20',
+                'rounded border-b-2 px-3 py-2 hover:bg-accent-primary',
```

Comment:
```suggestion
                'rounded border-b-2 px-3 py-2 hover:bg-controls-accent-hover',
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/9f12beb4-4bff-4d4f-9e88-c7880129f85a)

Reply:
This is slightly different element, not just control for which we have defined colors :(